### ISSUE TITLE
mongochemdeploy/ansible/roles/girder/tasks/main.yml': line 1, column 3

### ISSUE THREAD

**Issue by muammar** [OpenChemistry/mongochemdeploy#39]
tl;dr: The `monogchem` user is not being created and I don't know why that is. Google does not help me very much.  `ansible` requirements are installed but still the problem happens. Any idea why? I would be glad if anyone could shed some light (I am new to ansible). 

I am trying to setup a local environment to use mongochemdeploy and am facing some problems. I first proceeded to create some SSH keys to populate my `~/.ssh`. I installed `ansible` using `pip3` because this is a test in macOS. 

When I try 

```
± % ansible-playbook -i ansible/inventory/production ansible/site.yml -e=/Users/muammar/.ssh/
```

 I get the following error:

```
± % ansible-playbook -i ansible/inventory/production ansible/site.yml -e /Users/muammar/.ssh/                                                                                                                                                                            !800
 [WARNING]: Unable to parse /Users/muammar/git/mongochemdeploy/ansible/inventory/production as an inventory source

 [WARNING]: No inventory was parsed, only implicit localhost is available

 [WARNING]: provided hosts list is empty, only localhost is available. Note that the implicit localhost does not match 'all'

[DEPRECATION WARNING]: docker is kept for backwards compatibility but usage is discouraged. The module documentation details page may explain more about this rationale.. This feature will be removed in a future release. Deprecation warnings can be disabled by setting
deprecation_warnings=False in ansible.cfg.
ERROR! no action detected in task. This often indicates a misspelled module name, or incorrect module path.

The error appears to have been in '/Users/muammar/git/mongochemdeploy/ansible/roles/girder/tasks/main.yml': line 1, column 3, but may
be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:


- name: Create Girder monogchem user
  ^ here
```

Then I checked if I had the requirements correctly installed:

````
muammar@cs-it-7000008 ~/git/mongochemdeploy [devmekr *]
± % ansible-galaxy install -p ansible/roles/ -r ansible/requirements.yml                                                                                                                                                                                                 !805
- Stouts.mongodb (2.3.4) is already installed, skipping.
 [WARNING]: - geerlingguy.nginx (2.6.2) is already installed - use --force to change version to unspecified

 [WARNING]: - geerlingguy.certbot (3.0.2) is already installed - use --force to change version to unspecified

 [WARNING]: - girder.girder (v0.1) is already installed - use --force to change version to unspecified

- angstwad.docker_ubuntu (v3.5.0) is already installed, skipping.
````

**Comment by cjh1** [comment]
Hi @muammar, 

So I assume you where following the instructions in the README.md, I am afraid they are a little out of date we will get them updates. We have made the deployment much more straight forward, [here](https://github.com/OpenChemistry/mongochemdeploy/blob/master/dev/README.md) is a better place to start. Let use know how you get on.

**Comment by cjh1** [comment]
#40

**Comment by muammar** [comment]
@cjh1 your assumption is correct :) thanks for pointing me out to the right `README.md` file. I will follow-up once I have tried with those instructions.

**Comment by muammar** [comment]
@cjh1 I can confirm that the instructions you pointed me out to worked perfectly. I have my local version up and running :). Thanks.

**Comment by cjh1** [comment]
@muammar Excellent, that is great to hear. I will close this for now.
